### ISSUE TITLE
differences between cyclegan and discogan

### ISSUE THREAD

**Issue by nashory** [nashory/gans-collection.torch#1]
(https://www.quora.com/What-is-the-difference-between-CycleGAN-and-DiscoGAN-They-both-seem-to-be-the-same-thing)

+ CycleGAN has a single cycle-consistency loss. Also, when comparing F(G(y)) with y, it seems that the two papers use different forms of distance measures (MSE, hinge-loss).
+ CycleGan has an addition hyperparameter to adjust the contribution of reconstruction/cycle-consistency loss in the overall loss function.
+  the generator structures used in DiscoGan and CycleGan are somewhat different.

**Comment by Yash-5** [comment]
> CycleGAN has a single cycle-consistency loss.

Hey, about the first point, isn't the single cycle-consistency loss defined as a sum of the two forward cycle-consistent loss and backward cycle-consistent loss just as in DiscoGAN?

**Comment by nashory** [comment]
@Yash2208 yes, exactly. cyclegan uses one cycle-consistency losses by adding up the losses from each domain transfer generator (Gab + Gba) with multiplication of scale factor gamma (10 in the paper.), while DIscoGAN uses each losses respectively without summation.
I found this difference is very tirivial, though. :)

**Comment by doantientai** [comment]
The differences between 2 papers seem not enough to split them into 2 papers. I think they were submitted at quite the same time so they both got published. However, CycleGAN seems got more attention.
